validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -140,6 +140,7 @@ def finalize_block(self, block_header):
 class BlockVerifier(BlockVerifierInterface):
     """DevMode BlockVerifier implementation
     """
+    # pylint: disable=useless-super-delegation
```

Comment:
Rather than disable these, can we update the Interface classes to have concrete constructors and then just delete the `__init__`  methods here?